packages/dlog/src/dlog.ts

```diff
@@ -140,11 +142,57 @@ export interface DLogOpts {
 
 const allDlogs: Map<string, DLogger> = new Map()
 
+const isDev = typeof process !== 'undefined' && process.env.NODE_ENV !== 'production'
 // github#722
 const isSingleLineLogsMode =
     typeof process !== 'undefined' && process.env.SINGLE_LINE_LOGS === 'true'
 
-const makeDlog = (d: Debugger, opts?: DLogOpts): DLogger => {
+export const winstonLogger = createLogger({
+    exitOnError: false,
+    transports: [
+        ...(isDev
+            ? [
+                  new transports.Console({
+                      forceConsole: true,
+                      format: format.combine(
+                          format.colorize(),
+                          format.splat(),
+                          format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+                          format.printf(
+                              (info) =>
+                                  `${info.timestamp} ${info.level} [${info.label}]: ${info.message}`,
+                          ),
+                      ),
+                  }),
+                  new transports.Console({
+                      level: 'debug',
+                      forceConsole: true,
+                      format: format.combine(
+                          format.colorize(),
+                          format.splat(),
+                          format.timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+                          format.printf(
+                              (info) =>
+                                  `${info.timestamp} ${info.level} [${info.label}]: ${info.message}`,
+                          ),
+                      ),
+                  }),
+              ]
+            : [
+                  new transports.File({
+                      level: 'debug',
+                      filename: 'debug.log',
+                      format: format.combine(format.splat(), format.json()),
+                  }),
+                  new transports.File({
+                      filename: 'combined.log',
+                      format: format.combine(format.splat(), format.json()),
+                  }),
+              ]),
+    ],
```

Comment:
Should I add `transport.Console` here too? This way we can slowly migrate to file, but we can assure that it works with console first.

Reply:
ill do it